pkg/rpm/salt-minion

```diff
@@ -116,8 +116,16 @@ start() {
     # $1 - config dir
     local retval=0
 
+    if [ -f $LOCK_FILE ]; then
+        echo "Service $SERVICE:$MINION_USER:$MINION_ID is starting"
```

Comment:
What happens if for some reason the minion crashes and the lockfile is not cleaned up? Won't the minion then fail to start correctly until it is manually removed?

Reply:
This lock file is different from the usual usage and it only exist within the startup process.
When the startup is completed, whether the result is successful or fails, the lock file will be cleaned up.

Reply:
I don't like this either, it opens us up for problems later if the script is modified and the lockfile handling is not properly managed.

Reply:
hmm, seems in order to solve a small probability problem, a large probability problem is introduced. Before file lock management becomes reliable, i closed it.

Reply:
@terminalmage  @cachedout 
Now it seems that the problem can be solved.

Reply:
Sorry for delay in reviewing this, but the changes should be made in the salt-pack repository, see https://github.com/saltstack/salt-pack/blob/develop/file_roots/pkg/salt/2017_7/rhel6/sources/salt-minion for the 2017.7 branch and there is a similar change for the 2018.3 branch.

The contents of salt-minion are similar but different.  Need to update salt repository with a more recent version of packaging or pointers to salt-pack (salt-pack-py3 for Python3).

Further to the problem of cleanup on failure, perhaps should have a cleanup function added based off trapping signals, etc. which would ensure the lock file is removed.